core/src/main/java/org/elasticsearch/script/ScriptParameterParser.java

```diff
@@ -59,7 +53,7 @@ public ScriptParameterParser(Set<String> parameterNames) {
             fileParameters = new HashSet<>();
             indexedParameters = new HashSet<>();
             for (String parameterName : parameterNames) {
-                if (ScriptService.SCRIPT_LANG.match(parameterName)) {
+                if (ScriptService.SCRIPT_LANG.match(parameterName, ParseField.EMPTY_FLAGS)) {
```

Comment:
yes we could but it is not needed given what we are doing we did, in this specific case we are only interested in the return value of the call, cause if it's true we throw exception ourselves. Makes sense?